ui/src/app/job-list/table/table.component.html

```diff
@@ -41,23 +41,38 @@
       </button>
       <mat-menu #labelMenu="matMenu" class="details-menu">
         <mat-card>
-          <mat-card-title>{{ j.name }}</mat-card-title>
-          <p>{{ j.status }}</p>
-          <p>Submitted: {{ j.submission | jmShortDateTime }}</p>
+          <mat-card-subtitle>{{ j.status }}</mat-card-subtitle>
+          <mat-card-subtitle>Submitted: {{ j.submission | jmShortDateTime }}</mat-card-subtitle>
+          <mat-divider></mat-divider>
+          <mat-card-actions>
+            <mat-list>
+              <mat-list-item>
+                <a [routerLink]="[j.id]" [queryParams]="getQueryParams()">
+                  <clr-icon shape="arrow" dir="right" size="24"></clr-icon>
+                  <span>Job details</span>
+                </a>
+              </mat-list-item>
+              <mat-list-item *ngIf="canAbort(j)">
+                <a href="#" (click)="abortJob(j)">
+                  <clr-icon shape="times-circle" size="24"></clr-icon>
+                  <span>Abort job</span>
+                </a>
+              </mat-list-item>
+              <mat-list-item *ngIf="hasTimingUrl(j)">
+                <a href="{{ j.extensions.timingUrl }}" target="_blank">
+                  <clr-icon shape="hourglass" size="24"></clr-icon>
+                  <span>Timing Diagram</span>
+                </a>
+              </mat-list-item>
+              <mat-list-item *ngIf="j.name">
+                <a [href]="'?q=name%3D' + j.name">
+                  <clr-icon shape="filter-2" size="24"></clr-icon>
+                  <span>Filter by <strong>name:{{ j.name }}</strong></span>
```

Comment:
I'm not sure I follow the product thinking here. Why are we expediting "filter by name" here vs other criteria? Is this because the button happens to hang off of the job name column?

Reply:
The ability to filter the job list by the `key:value` pair for a particular cell in the job table has been a proposed UI idea (see https://github.com/DataBiosphere/job-manager/issues/340 and
https://github.com/DataBiosphere/job-manager/issues/319, for example).  Because the `name` column already has a menu attached to it, it is easier to add it here first.  Eventually, the idea is that this will be something the user can do anywhere in the table.